functions/src/index.ts

```diff
@@ -47,3 +48,11 @@ export const importCsv = onHttpsRequest(importCsvHandler);
 export const importGeoJson = onHttpsRequest(importGeoJsonHandler);
 
 export const exportCsv = onHttpsRequest(exportCsvHandler);
+
+export const onWriteSurvey = functions.firestore
+  .document('surveys/{surveyId}')
+  .onWrite(onWriteSurveyHandler);
+
+export const onWriteLoi = functions.firestore
+  .document('surveys/{surveyId}/lois/{loiId}')
```

Comment:
@os-micmec We should probably refactor these paths into exports of `on-write-survey.ts` so that dependency on `{surveyId}` string is defined in one place.